RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Bundle.properties

```diff
@@ -105,3 +105,8 @@ SearchEngineURLQueryAnalyzer.toString=Name\: {0}\n\
 SearchEngineURLQueryAnalyzer.parentModuleName.noSpace=RecentActivity
 SearchEngineURLQueryAnalyzer.parentModuleName=Recent Activity
 UsbDeviceIdMapper.parseAndLookup.text=Product\: {0}
+ExtractEdge.moduleName=Microsoft Edge
+ExtractEdge.process.errMsg.unableFindESEViewer=Unable to find ESEDatabaseViewer
+ExtractEdge.process.errMsg.errGettingWebCacheFiles=Error retrieving Edge file
+ExtractEdge.process.errMsg.noWebCachFiles=No Edge WebCache file found
+ExtractEdge.process.errMsg.errWriteFile={0}\: Error while trying to write file\:{1}
```

Comment:
I noticed that in case.java the message annotation was before each function.  Is there a reason for that?  If there is one that is repeatedly used can I define them at the top of the file?

Reply:
Yes, I think that you can. It certainly would be sensible to do so.